src/main/java/fr/cgi/magneto/controller/CardController.java

```diff
@@ -68,14 +71,17 @@ public void getAllCardsByBoardId(HttpServerRequest request) {
         UserUtils.getUserInfos(eb, request, user -> {
             String boardId = request.getParam(Field.BOARDID);
             Integer page = request.getParam(Field.PAGE) != null ? Integer.parseInt(request.getParam(Field.PAGE)) : null;
-            cardService.getAllCardsByBoard(user, boardId, page, false)
-                    .onSuccess(result -> renderJson(request, result))
-                    .onFailure(fail -> {
-                        String message = String.format("[Magneto@%s::getAllCardsByBoardId] Failed to get all cards : %s",
-                                this.getClass().getSimpleName(), fail.getMessage());
-                        log.error(message);
-                        renderError(request);
-                    });
+            boardService.getBoards(Collections.singletonList(boardId))
+                    .compose(board ->
```

Comment:
Attention, ton success est à l'intérieur de ton compose
ca devrait être au même niveau : 

```suggestion
      boardService.getBoards(Collections.singletonList(boardId))
                    .compose(board -> cardService.getAllCardsByBoard(user, board.get(0), page, false))
                    .onSuccess(result -> renderJson(request, result))
                    .onFailure(fail -> {
                        String message = String.format("[Magneto@%s::getAllCardsByBoardId] Failed to get all cards : %s",
                                this.getClass().getSimpleName(), fail.getMessage());
                        log.error(message);
                        renderError(request);
                    });
```